### ISSUE TITLE
vk_layer_validation_tests generate out-of-test DS errors

### ISSUE THREAD

**Issue by mark-lunarg** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1031]
Running the LVTs will result in 8 or 10 DrawState errors which are generated during the framework teardown after a few of the tests.  These fall outside the scope of the ErrorMonitor and so do not cause actual LVT failures

DS(ERROR): object: 0x5 type: 14 location: 5479 msgCode: 75: Cannot delete image view 0x5 that is currently in use by a command buffer.
DS(ERROR): object: 0x2 type: 10 location: 5479 msgCode: 75: Cannot delete image 0x2 that is currently in use by a command buffer.

**Comment by tobine** [comment]
Note that the general solution to these issues is, prior to test completion, add a vkQueueWaitIdle() on any queues where cmd buffers were submitted

**Comment by Tony-LunarG** [comment]
I don't think that's all that's going on here.  I picked one of the tests reporting the errors, CommandBufferTwoSubmits, and fixed it to wait for a fence between submits, and to vkDeviceWaitIdle at the end, and it's still claiming that the renderpass, comand buffer, image view and image are all still in use even after DeviceWaitIdle.  Going back to commit d45139791  the renderpass, image view and image errors all go away, so I think the recent rework of the in_use checks may have a problem, especially with multiple submits followed by a vkDeviceWaitIdle. Old and new code both mistakenly think the command buffer is still in use after vkDeviceWaitIdle.  I've pushed my test changes on a tony-in-use-test branch.
